.gitignore

```diff
@@ -1,3 +1,4 @@
+!building_blocks/
```

Comment:
Why this change?

Reply:
Our `.gitignore` file includes `build*/`. That would also ignore all `building_blocks` folders. I added this line so `building_blocks` are exempt from ignoring.

Reply:
Woops >_>

Reply:
You're kind of right. Modifying it to `/build*/` should limit it to folders from root folder only.
I'll fix that.